helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -888,6 +885,91 @@ public Response updateCloudConfig(@PathParam("clusterId") String clusterId,
     return OK();
   }
 
+  /**
+   * Get the global target task thread pool size of the cluster, a value that's used to construct
+   * task thread pools for the cluster's instances and is created by users. This value is used
+   * only if the instances don't specify their own target thread pool sizes in their
+   * InstanceConfigs; if this value needs to be used but is not specified, a default value -
+   * TaskConstants.DEFAULT_TASK_THREAD_POOL_SIZE will be used instead.
+   * @param clusterId - the cluster id of which the global target thread pool size belongs to
+   */
+  @GET
+  @Path("{clusterId}/global-target-task-thread-pool-size")
```

Comment:
To make it more generic, how about

`"{clusterId}/task-framework/thread-pool-size/"`? @jiajunwang @dasahcc  what do you think?

This way, we open up a namespace for other future task framework-related configs.

Reply:
Do we have namespace for WAGED or some other components? If yes, I am OK to have it now. Otherwise, we need to refactor the URL structure and how to keep compatibility will be a problem.